pyglmnet/utils.py

```diff
@@ -130,3 +135,22 @@ def set_log_level(verbose):
             raise ValueError('verbose must be of a valid type')
         verbose = logging_types[verbose]
     logger.setLevel(verbose)
+
+
+def return_verbose_iterable(data):
```

Comment:
We can basically wrap any list/range object with tqdm with this method. It depends on the logger's level, this means that if the user has set `verbose=True`, then he will see the progress bar. This way we don't need any new state variable (e.g., `show_progress`).